app/controllers/hiring_staff/vacancies_controller.rb

```diff
@@ -16,7 +16,7 @@ def new
   end
 
   def edit
-    vacancy = current_school.vacancies.find(id)
+    vacancy = current_school.vacancies.find(vacancy_id)
```

Comment:
I could change `vacancy_id` across this file into `job_id` to match the param's name instead of matching the ruby object's name. This would be a trade-off between two different sorts of consistency. Which is preferred?

Reply:
Personally, I'd rather it matched the Ruby object's name. However, @stevenleggdfe is likely to be the person most affected by changes in parameters.